components/pages/Main.js

```diff
@@ -30,45 +31,48 @@ function ColoredIcon(name, color) {
 }
 
 function TabNavigator() {
-  const user = useSelector((state) => state.user).data;
   return (
-    user !== undefined && (
-      <Tab.Navigator>
-        <Tab.Screen
-          name='Home'
-          component={Home}
-          options={{
-            tabBarLabel: 'Home',
-            tabBarIcon: ({ color }) => ColoredIcon('home', color),
-          }}
-        />
-        <Tab.Screen
-          name='Profile'
-          component={Profile}
-          initialParams={{ userId: getAuth().currentUser.uid }}
-          options={{
-            tabBarLabel: 'Profile',
-            tabBarIcon: ({ color }) => ColoredIcon('bell', color),
-          }}
-        />
-        <Tab.Screen
-          name='CommunityTab'
-          component={CommunityTab}
-          options={{
-            tabBarLabel: 'Communities',
-            tabBarIcon: ({ color }) => ColoredIcon('account-group', color),
-          }}
-        />
-        <Tab.Screen
-          name='Messages'
-          component={DMList}
-          options={{
-            tabBarLabel: 'Messages',
-            tabBarIcon: ({ color }) => ColoredIcon('chat', color),
-          }}
-        />
-      </Tab.Navigator>
-    )
+    <Tab.Navigator
+      options={{
+        headerBackVisible: false,
+      }}
+    >
+      <Tab.Screen
+        name='Home'
+        component={Home}
+        options={{
+          tabBarLabel: 'Home',
+          tabBarIcon: ({ color }) => ColoredIcon('home', color),
+          headerShown: false,
```

Comment:
why the headerShown: false here and on community tab but not on the other two tabs? seems to behave as if there's no header in general.